Sources/Widgets/Representations/SphereHandleRepresentation/index.js

```diff
@@ -132,6 +132,10 @@ function vtkSphereHandleRepresentation(publicAPI, model) {
         (!state.isVisible || state.isVisible() ? 1 : 0) *
         (state.getScale1 ? state.getScale1() : model.defaultScale);
 
+      if (publicAPI.getScaleByDisplay()) {
+        typedArray.scale[i] *= publicAPI.getDisplayWorldHeightAt(coord);
```

Comment:
I wonder if using `center` is not more reliable than `origin`

Reply:
What do you mean by `center`?

Reply:
center of the `bounds` vs `origin` of the `bounds`.

Reply:
`origin` is assumed to be the center of the sphere handle, which should match with the center of the sphere bounds.

Reply:
Fair enough